client/blocks/post-item/index.jsx

```diff
@@ -145,19 +146,27 @@ class PostItem extends React.Component {
 								</a>
 							) }
 							{ ! isPlaceholder &&
-							externalPostLink && (
-								<ExternalLink
-									icon={ true }
-									href={ postUrl }
-									target="_blank"
-									className="post-item__title-link"
-								>
-									{ title || translate( 'Untitled' ) }
-								</ExternalLink>
-							) }
+								externalPostLink && (
+									<ExternalLink
+										icon={ true }
+										href={ postUrl }
+										target="_blank"
+										className="post-item__title-link"
+									>
+										{ title || translate( 'Untitled' ) }
+									</ExternalLink>
+								) }
 						</h1>
 						<div className="post-item__meta">
 							<PostTime globalId={ globalId } />
+							{ post && (
+								<PostActions
+									className={ 'post-item__post-actions' }
```

Comment:
We can just use `""` instead of `{ '' }` here.

Reply:
👍 Updated.

Reply:
It appears they are actually being changed by prettifier when I attempt to commit.

Reply:
Ugh.

Reply:
Ignore me -- I'm having problems parsing English this morning :)